using_images/s2i_images/java.adoc

```diff
@@ -85,14 +86,19 @@ You can override these default goals and options by specifying the following env
 
 |Variable name |Description
 
-|`*ARTIFACT_DIR*`
-|The relative path to the target where JAR files are created for multi-module builds.
+|`*MAVEN_S2I_ARTIFACT_DIRS*`
+|Relative paths of source directories to scan for build output, which will be copied to $DEPLOY_DIR. Defaults to **target**.
 
 |`*JAVA_MAIN_CLASS*`
 |The main class to use as the argument to Java. This can also be specified in the *_.s2i/environment_* file as a Maven property inside the project (*docker.env.Main*).
```

Comment:
@jmtd Should this information be removed? You added the information below with your last review, but this information was not removed but seems similar.

Reply:
I removed ARTIFACT_DIR because it has been deprecated, and MAVEN_S2I_ARTIFACT_DIRS should be used instead. I added MAVEN_ARGS_APPEND which supplements the existing MAVEN_ARGS. I didn't remove MAVEN_ARGS or JAVA_MAIN_CLASS because they are still valid.

Reply:
@jmtd I referring to the information that starts on line 93. The description for JAVA_MAIN_CLASS has two similar paragraphs. I was wondering if I should remove: "The main class to use as the argument to Java. This can also be specified in the *_.s2i/environment_* file as a Maven property inside the project (*docker.env.Main*)."

Reply:
Thanks for clarifying! Yes that line should indeed go, and the empty line after it I suppose too, leaving the line beginning "A main class" which will need "|" pre-pended.